pages/calendar/index.js

```diff
@@ -161,23 +157,10 @@ const CalendarDemo = () => {
                             <label htmlFor="touchUI">TouchUI</label>
                             <Calendar id="touchUI" value={date13} onChange={(e) => setDate13(e.value)} touchUI />
                         </div>
-                        <div className="field col-12 md:col-4">
-                            <label htmlFor="mask">Mask</label>
-                            <Calendar id="mask" value={date14} onChange={(e) => setDate14(e.value)} mask="99/99/9999" />
```

Comment:
@mcandu Why did you remove these demos? They were added based on user feedback and to show extra features that differ from other libraries.

Reply:
@mertsincan Because PrimeNG and PrimeVue demos are sync. I thought PrimeReact demos should be the same like these.